perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -256,9 +271,16 @@ public void setSendingEnabled(PerunSession sess, List<ApplicationMail> mails, bo
 		for (ApplicationMail mail : mails) {
 			// update sending (enabled / disabled)
 			if (Compatibility.isPostgreSql()) {
-				jdbc.update("update application_mails set send=? where id=?", (enabled) ? "1" : "0", mail.getId());
+				try {
+					jdbc.update("update application_mails set send=? where id=?", (enabled) ? "1" : "0", mail.getId());
```

Comment:
``jdbc.update()`` doesn't throws ``EmptyResultDataAccessException`` since it doesn't select the result, but it only returns number of affected rows. You should do the same, as in Oracle DB case below.
Also you should wrap calls ``jdbc.update()`` in try/catch to cache ``RuntimeException`` and wrap it in our ``InternalErrorException``. It should be done everywhere, but for the purpose of this pull-request it will be sufficient to fix it in modified methods.

Reply:
My bad, it should be fixed now.